ui/etc/schema.yml

```diff
@@ -87,7 +87,7 @@ mapping:
 
       replication_group_defaults:
         type: map
-        required: True
+        allowempty: True
```

Comment:
Okay.  I followed the way the other optional maps in the schema appeared to be set up.  Maybe review those to make sure they are also `required: False` rather than `allowempty: True`?